### ISSUE TITLE
Show that Margin is always the best AL method when fine-tuning SSL models

### ISSUE THREAD

**Issue by luiscarlosgph** [luiscarlosgph/jiayuan-huang-phd#32]
- Show it for different datasets
- Show it for different downstream tasks

**Comment by PeterHJY628** [comment]
- finish the val loss of SSL experiments to see if the transition point between AL and RS is highly related to the loss decrease transition point
- Run the transition experiments to see if it is better than just random or just AL. Compare it with dynamic AL

**Comment by PeterHJY628** [comment]
[0430.pptx](https://github.com/user-attachments/files/27276012/0430.pptx)
